integration-tests/src/test/resources/sitconfig/java/SitConfigTests.java

```diff
@@ -0,0 +1,522 @@
+              adminPassword,
+              DomainRuntimeServiceMBean.MBEANSERVER_JNDI_NAME);
+      editMbs =
+          lookupMBeanServerConnection(
+              adminHost,
+              adminPort,
+              adminUser,
+              adminPassword,
+              EditServiceMBean.MBEANSERVER_JNDI_NAME);
+
+      ObjectName runtimeserviceObjectName = new ObjectName(RuntimeServiceMBean.OBJECT_NAME);
+      runtimeServiceMBean =
+          (RuntimeServiceMBean)
+              MBeanServerInvocationHandler.newProxyInstance(runtimeMbs, runtimeserviceObjectName);
+
+      ObjectName domainServiceObjectName = new ObjectName(DomainRuntimeServiceMBean.OBJECT_NAME);
+      domainServiceMBean =
+          (DomainRuntimeServiceMBean)
+              MBeanServerInvocationHandler.newProxyInstance(
+                  domainRuntimeMbs, domainServiceObjectName);
+
+      ObjectName editserviceObjectName = new ObjectName(EditServiceMBean.OBJECT_NAME);
+      editServiceMBean =
+          (EditServiceMBean)
+              MBeanServerInvocationHandler.newProxyInstance(editMbs, editserviceObjectName);
+    } catch (MalformedObjectNameException ex) {
+      Logger.getLogger(SitConfigTests.class.getName()).log(Level.SEVERE, null, ex);
+      throw new AssertionError(ex.getMessage());
+    }
+  }
+
+  private MBeanServerConnection lookupMBeanServerConnection(
+      String host, String adminPort, String user, String adminPassword, String jndiName)
+      throws MalformedURLException, IOException, Exception {
+    JMXServiceURL serviceURL;
+    MBeanServerConnection mBeanServerConnection;
+
+    println(
+        "Host: "
+            + adminHost
+            + " Port: "
+            + adminPort
+            + " adminUser: "
+            + adminUser
+            + " adminPassword :"
+            + adminPassword);
+    String protocol = "t3";
+    Integer portInteger = Integer.valueOf(adminPort);
+    int port = portInteger;
+    HashMap h = new HashMap();
+    h.put(Context.SECURITY_PRINCIPAL, adminUser);
+    h.put(Context.SECURITY_CREDENTIALS, adminPassword);
+    h.put(JMXConnectorFactory.PROTOCOL_PROVIDER_PACKAGES, "weblogic.management.remote");
+    h.put("jmx.remote.x.request.waiting.timeout", new Long(10000));
+    serviceURL = new JMXServiceURL(protocol, adminHost, port, "/jndi/" + jndiName);
+    println("Making mbean server connection with url");
+    println(serviceURL.toString());
+    jmxConnector = JMXConnectorFactory.connect(serviceURL, h);
+    mBeanServerConnection = jmxConnector.getMBeanServerConnection();
+    if (mBeanServerConnection == null) {
+      throw new Exception("MBean server connection is null");
+    }
+    return mBeanServerConnection;
+  }
+
+  protected void verifyDebugFlagJMXCore(boolean expectedValue) {
+    ServerMBean serverMBean = getServerMBean();
+    ServerDebugMBean serverDebugMBean = serverMBean.getServerDebug();
+    boolean debugFlag = serverDebugMBean.getDebugJMXCore();
+    assert expectedValue == debugFlag
```

Comment:
what does this mean?  a == b : x ??

Reply:
assert that a and b are equal, If not fail with message "x"